rules/build_sector.smk

```diff
@@ -212,17 +212,72 @@ rule build_temperature_profiles:
         "../scripts/build_temperature_profiles.py"
 
 
+rule build_central_heating_temperature_profiles:
+    params:
+        max_forward_temperature_central_heating=config_provider(
+            "sector",
+            "district_heating",
+            "supply_temperature_approximation",
+            "max_forward_temperature",
+        ),
+        min_forward_temperature_central_heating=config_provider(
+            "sector",
+            "district_heating",
+            "supply_temperature_approximation",
+            "min_forward_temperature",
+        ),
+        return_temperature_central_heating=config_provider(
+            "sector",
+            "district_heating",
+            "supply_temperature_approximation",
+            "return_temperature",
+        ),
+        snapshots=config_provider("snapshots"),
+        lower_threshold_ambient_temperature=config_provider(
+            "sector",
+            "district_heating",
+            "supply_temperature_approximation",
+            "lower_threshold_ambient_temperature",
+        ),
+        upper_threshold_ambient_temperature=config_provider(
+            "sector",
+            "district_heating",
+            "supply_temperature_approximation",
+            "upper_threshold_ambient_temperature",
+        ),
+        rolling_window_ambient_temperature=config_provider(
+            "sector",
+            "district_heating",
+            "supply_temperature_approximation",
+            "rolling_window_ambient_temperature",
+        ),
```

Comment:
This is perfectly alright, but one could shorten this by using a whole config block and retrieving the individual settings from `snakemake.params.supply_temperature_approximation[attr]`. Not necessary to adjust.
```suggestion
        supply_temperature_approximation=config_provider(
            "sector",
            "district_heating",
            "supply_temperature_approximation",
        ),
        snapshots=config_provider("snapshots"),
```